cylc/uiserver/scripts/hub.py

```diff
@@ -24,7 +24,7 @@
 
 from cylc.uiserver import (
     __version__,
-    __file__ as uis_pkg
+    __file__ as uis_pkg,
```

Comment:
Any particular reason?

Reply:
No, I initially added init_log to this import. I'll remove.

Reply:
FYI: trailing commas [perfectly valid] are a signal to `black` to leave room for further additions so can be considered functional.

Reply:
So do I; it was more that I was curious that this had been changed at all for this PR.